vep/src/main/resources/listVariants.py

```diff
@@ -75,6 +75,7 @@ def main():
     # output the variants as CSV part files
     df.write \
         .mode('overwrite') \
+        .option("compression", "org.apache.hadoop.io.compress.ZStandardCodec") \
```

Comment:
I think this is the stage that provides inputs to VEP, we want it to leave zst files in s3.